sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -64,6 +64,7 @@ public static IOpenTelemetryBuilder UseAzureMonitorExporter(this IOpenTelemetryB
                 builder.Services.Configure(configureAzureMonitor);
             }
 
+            // Note: We automatically turn on signals for "UseOtlpExporter"
```

Comment:
This comment doesn't meet my expectations.

1) This isn't the OTLP Exporter :)
2) I think it's confusing to a reader why lines 68-71 only set a MetricExporter but nothing for Logs or Traces. I'd like to see a brief statement explaining the strategy of using the ExporterRegistrationHostedService to add the other exporters while also resolving some other components from the service collection.

Reply:
Did you check - https://github.com/Azure/azure-sdk-for-net/blob/3b8ce6028d5ab7f32a9d25c3be639c2485e66db2/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/ExporterRegistrationHostedService.cs#L44

I'm not in favor of adding all the directions in the code comment, a similar strategy is followed in OpenTelemetry .NET too.

Will fix UseOtlpExporter

Reply:
Since your expectations differ, could you please suggest the details you would like to include here?